js/how-does-javascript-work/how-does-javascript-work.md

```diff
@@ -0,0 +1,149 @@
+Hello, 👋 this is Shivam Verma, this is my first blog and in this, I will help you to understand the working of JavaScript. Let's Begin!!
+
+#### Ever wondered how JavaScript works under the hood?     
```

Comment:
There is a meaning to number of `#` we use in articles

`#` => H1
`##` => H2
`###` => H3 and so on

What you're trying to say looks like subheading (since the blog title will be heading: H1)

You can use `##` here 

And if something is nested inside sub heading, means sub sub heading, that will be represented by `###`

The number of `#` depends on nesting